src/vkscript/parser.cc

```diff
@@ -204,7 +257,12 @@ Result Parser::ProcessShaderBlock(const SectionParser::Section& section) {
   shader->SetFormat(section.format);
   shader->SetData(section.contents);
 
-  Result r = script_->AddShader(std::move(shader));
+  Result r = script_->GetPipeline(kDefaultPipelineName)
```

Comment:
Here the pipeline name is hard-coded as `kDefaultPipelineName`.
Does it mean we will have only the default pipeline?

Reply:
I have a feeling vkscript will only have one pipeline, similar to how it works now. AmberScript will have multiple pipelines.